lab05-2.cpp

```diff
@@ -163,57 +163,65 @@ void runHomographFunction(const string path1, const string path2)
  *************************/
 void runTestCases()
 {
-    string path1, path2;
+    string path1, path2, current_path = "C:users/home/secret";
     
+    cout << "Base path: " << current_path << endl << endl;
+
     cout << "# CASE 1: Same directory #" << endl;
-    path1 = "C:user/home/secret/password.txt";
-    path2 = "./pasword.txt";
+    path1 = "C:users/home/secret/password.txt";
+    path2 = "./password.txt";
     cout << "   path 1: " << path1 << endl;
     cout << "   path 2: " << path2 << endl;
-    cout << "   > The paths are homographs\n\n";
+    runHomographFunction(path1, path2, current_path);

```

Comment:
I think we need to actually use runHomographFunction in our tests, otherwise they aren't testing anything